cli/snapshot.go

```diff
@@ -15,6 +17,42 @@ import (
 	"github.com/mendersoftware/mender/utils"
 )
 
+func (runOpts *runOptionsType) DumpSSH(ctx *cli.Context) error {
+	var identitySet bool
+	numArgs := ctx.NArg()
+	args := make([]string, numArgs)
+	for i := 0; i < numArgs; i++ {
+		args[i] = ctx.Args().Get(i)
+		if args[i] == "-i" {
+			identitySet = true
+		}
+	}
+	if !identitySet {
+		// If called with `sudo` set identity to caller default
+		uid := os.Getenv("SUDO_UID")
+		usr, err := user.LookupId(uid)
+		if err == nil {
+			args = append(args, "-i", usr.HomeDir+"/.ssh/id_rsa")
```

Comment:
This file is not guaranteed to exist, and in fact, it's quite unlikely on an embedded device. In this case I assume that the command will fail. Whether or not it will work out of the box depends on policy, so I think I would consider adding this as a suggestion to the command help text instead of doing anything with it here.